ember-app/app/components/hb-milestone-card.js

```diff
@@ -1,7 +1,43 @@
 import HbCard from "../components/hb-task-card";
+import Ember from "ember";
+
+var { get, set } = Ember;
+
+var takeWhile = function(list, callback, context) {
```

Comment:
we aren't using lodash unfortunately. 

We could upgrade to lodash at some point, I think I lifted this method from a gist I found of someones common underscore extensions